### PR TITLE
Revise baseline loading

### PR DESCRIPTION
**Pull request by rod-glover**
Data values coded -9999 are absent and should not be loaded in the database.

Possibly some precip data should not be loaded (alternatively, it should be loaded and filtered out of the relevant queries later). Pending response from Faron.

Test code tightened up somewhat.

### CODE REVIEW

**Comment by rod-glover** [comment]
Resolution on loading exclusion: Stations specified in an exclusion file can now be excluded. Later it turned out this is unnecessary, since all stations to be explicitly excluded are not present in the database currently, and the loading script already skipped/excluded any such non-existent stations encountered in the input file. Leaving in the exclusion feature since it is innocuous and may be useful at some future date.

File: pycds/climate_baseline_helpers.py [hunk 1]
```diff
@@ -140,40 +155,58 @@ def parse_line(line):
...
+                            DerivedValue(
+                                time=datetime.datetime(baseline_year, month, baseline_day(month), baseline_hour),
+                                datum=convert(datum),
+                                variable=variable,
+                                history=latest_history
+                            )
+                        )
+                n_added += 1
+            else:
+                logging.info('Skipping input line:')
```
Comment (jameshiebert) [inline pycds/climate_baseline_helpers.py]: Looks like good, informative logging overall. Thanks for making sure that the user knows what's going on.

File: scripts/load-climate-baseline-values.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
     parser.add_argument("-d", "--dsn", help="Database DSN in which to create new network")
     parser.add_argument("-v", "--variable", help="Name of variable to be loaded")
     parser.add_argument("-f", "--file", help="Path of file containing climate baseline values to be loaded")
+    parser.add_argument("-e", "--exclude", help="Path of file containing native ids of stations to be excluded from loading, one per line")
```
Comment (jameshiebert) [inline scripts/load-climate-baseline-values.py]: Good way to deal with this problem, even if we don't necessarily have to use it at first.

File: tests/test_climate_baseline_helpers.py [hunk 3]
```diff
@@ -57,23 +49,23 @@ def test_creates_the_expected_new_network_record(empty_database_session):
         assert result.publish == True
         assert 'PCIC' in result.long_name
 
-    def test_creates_no_more_than_one_of_them(empty_database_session):
-        sesh = empty_database_session
+    def it_creates_no_more_than_one_of_them(session):
+        sesh = session
```
Comment (jameshiebert) [inline tests/test_climate_baseline_helpers.py]: Again, the aliasing is probably not necessary in this case. Doesn't save that much space.

Comment (rod-glover) [inline tests/test_climate_baseline_helpers.py]: This is covered in PR #10. But it doesn't cover such cases, probably, and there are other little inconsistencies. How about we leave this aside and I will create a cleanup PR for the leftovers.

**Comment by rod-glover** [comment]
I'd be happy to know what other things you'd change. I'd probably learn from it.

**Comment by rod-glover** [comment]
Ready for next review

**Comment by jameshiebert** [comment]
Shoot, do you mind rebasing or resolving the conflicts?

**Comment by rod-glover** [comment]
Was just getting on that.

**Comment by rod-glover** [comment]
Ready

**Comment by jameshiebert** [comment]
Sweet!

**Pull request merged**